File-Formats/DocIO/Convert-Word-Document-to-Image-in-Azure-App-Service-Windows.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Convert Word document to Image in Azure App Service on Windows | Syncfusion
+description: Convert Word document to Image in Azure App Service on Windows using .NET Core Word (DocIO) library without Microsoft Word or interop dependencies.
+platform: file-formats
+control: DocIO
+documentation: UG
+---
+
+# Convert Word to Image in Azure App Service on Windows
+
+Syncfusion  DocIO is a [.NET Core Word library](https://www.syncfusion.com/document-processing/word-framework/net/word-library) used to create, read, edit and **convert Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, you can **convert a Word document to Image in Azure App Service on Windows**
+
+## Steps to convert Word document to Image in Azure App Service on Windows:
+
+Step 1: Create a new ASP.NET Core Web App (Model-View-Controller).
+![Create a ASP.NET Core Web App project](Azure_Images/App_Service_Linux/Create-Project-WordtoPDF.png)
+
+Step 2: Create a project name and select the location.
+![Configure your new project](Azure_Images/App_Service_Linux/Configure_Project_WordtoImage.png)
+
+Step 3: Click **Create** button
+![Additional Information](Azure_Images/App_Service_Linux/Additional_Information_WordtoPDF.png)
+
+Step 4: Install the [Syncfusion.DocIORenderer.Net.Core](https://www.nuget.org/packages/Syncfusion.DocIORenderer.Net.Core) NuGet package as a reference to your project from [NuGet.org](https://www.nuget.org/).
+
+![Install Syncfusion.DocIORenderer.Net.Core Nuget Package](Azure_Images/App_Service_Linux/Syncfusion_Nuget_Package_WordtoPDF.png)
+
+Step 5: Add a new button in the **Index.cshtml** as shown below.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+@{Html.BeginForm("WordToImage", "Home", FormMethod.Post, new { enctype = "multipart/form-data" });
+ 
```

Comment:
Remove unwanted empty lines in code snippet.

Reply:
updated